### PR TITLE
Add Kinesis API and fix some typo

### PR DESCRIPTION
**Pull request by 2mf**
add kinesis API:
 - AddTagsToStream
 - ListTagsForStream
 - RemoveTagsFromStream

fix some typo

### CODE REVIEW

File: moto/kinesis/responses.py [hunk 1]
```diff
@@ -24,7 +25,9 @@ def is_firehose(self):
     def create_stream(self):
         stream_name = self.parameters.get('StreamName')
         shard_count = self.parameters.get('ShardCount')
-        self.kinesis_backend.create_stream(stream_name, shard_count, self.region)
+        stream = self.kinesis_backend.create_stream(stream_name, shard_count, self.region)
+        if isinstance(stream, BadRequest):
```
Comment (spulec) [inline moto/kinesis/responses.py]: Is this just the result of a bug in `create_stream`? I think `create_stream` should be raising the exception.

**Comment by spulec** [comment]
Can you add a couple tests?

File: moto/kinesis/responses.py (hunk 1 above)
Comment (2mf) [inline moto/kinesis/responses.py]: You are right, it should raise a exception in `self.kinesis_backend.create_stream`.
I'll update the request

**Comment by 2mf** [comment]
Sure I'll add a couple tests, but what about the existing API ?

I'm currently working on implementing the `merge_shard` and `split_shard` thing including a scheme that all shards are used in a stream not just the first one but I'll make another pull-request for that in Dec.

**Comment by spulec** [comment]
Are you working on those additional endpoints on this branch? Happy to merge this now or wait for those.

**Comment by 2mf** [comment]
This branch is only for mentioned API's. It would be great if you could merge these changes for now :-)

**Comment by 2mf** [comment]
Ok, I implemented basic sharding, split_shard, merge_shards.
Testcases were added and/or adjusted.

**Comment by spulec** [comment]
Thanks for all of your work on this!

**Pull request merged**